docs/symboltable_serialization.md

```diff
@@ -0,0 +1,63 @@
+# Abstract
+In this document we are going to present how Jariko allows managing programs memory persistence using as approach the serialization.  
```

Comment:
not "how Jariko allows managing programs memory persistence using as approach the ." but "how Jariko uses serialization to manage programs' memory's persistence"

Reply:
ok